### DISCUSSION TITLE
Interactive user tutorial: 'Zen of CUE'

### DISCUSSION THREAD

**Discussion by ptMcGit** [cue-lang/cue#2221] [Ideas]
Since this is my first post I'd like to say thanks for this great language!

It took some time to get to my current level of understanding. I wrote down a bunch of examples along the way  to demonstrate how CUE works. I had an idea for how I could share these -- here is a simple POC/demo: https://github.com/ptMcGit/cue/tree/zen-of-cuelang

I would be happy to collaborate with some folks if this is of interest to the community / anyone.

**Comment by Lyoness** [comment]
@ptMcGit Zach, would love to collaborate. Can you email me at carmen@cue.works?  Perhaps this is something we can incorporate the rework of the website, beta.cuelang.org?

**Comment by demizer** [comment]
Big +1 here. CUE looks to be fantastic tool for our use case in packaging system software, but it's still early days. The learning curve for me has been a slog because it's different from anything else I've ever used. The official docs are good, but terse and lack deeper examples.

**Reply by Lyoness** [reply]
Thanks for this feedback, this has been heard loud and clear, and we're looking forward to changing that in the coming months.
